src/main/java/nl/tudelft/ti2206/group9/gui/ShopScene.java

```diff
@@ -0,0 +1,85 @@
+package nl.tudelft.ti2206.group9.gui;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.scene.Node;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ListView;
+import nl.tudelft.ti2206.group9.ShaftEscape;
+import nl.tudelft.ti2206.group9.level.State;
+import nl.tudelft.ti2206.group9.util.GameObserver;
+import static nl.tudelft.ti2206.group9.ShaftEscape.OBSERVABLE;
+
+/**
+ * Class that creates the content for a shop screen.
+ *
+ * Created by Maikel on 04/10/2015.
```

Comment:
Delete line 18 and change line 19 to @author.
Just for symmetry. ;)

<review,documentation